### PR TITLE
Switching to a single filter db for all samples

### PR DESCRIPTION
**Pull request by tanaes**
Switching to a single filter db, per issue #68 @mortonjt

### CODE REVIEW

**Comment by tkosciolek** [comment]
@jwdebelius you might want to take a look at this

**Comment by mortonjt** [comment]
any luck on this?

File: oecophylla/qc/qc.rule [hunk 1]
```diff
@@ -54,7 +54,7 @@ rule qc_filter:
         forward = qc_dir + "{sample}/filtered/{sample}.R1.trimmed.filtered.fastq.gz",
         reverse = qc_dir + "{sample}/filtered/{sample}.R2.trimmed.filtered.fastq.gz"
     params:
-        filter_db = lambda wildcards: config['samples'][wildcards.sample]['filter_db'],
+        filter_db = config['params']['filter_db'],
```
Comment (tkosciolek) [inline oecophylla/qc/qc.rule:57]: Travis reports an error here

**Review by tkosciolek — APPROVED** [review]

**Comment by jwdebelius** [comment]
:+1:

**Pull request merged**